### DISCUSSION TITLE
Authelia Active Directory Authentication stopped working

### DISCUSSION THREAD

**Discussion by tltglitch** [authelia/authelia#6595] [Q&A] [answered]
My authelia authentication stopped working. Nothing has changed in the environment but i am unable to login with an AD user. I receive the error below but the bind credentials are correct and have the permissions. I even used the admin user  to bind but still getting this error.

{"level":"error","method":"POST","msg":"Unsuccessful 1FA authentication attempt by user 'johnoliver': cannot find user DN of user 'johnoliver'. Cause: error occurred connecting to referred LDAP server 'ldaps://ForestDnsZones.corp.evilcorp.com/DC=ForestDnsZones,DC=corp,DC=evilcorp,DC=com': dial failed with error: LDAP Result Code 200 \"Network Error\": dial tcp: lookup ForestDnsZones.corp.evilcorp.com: no such host","path":"/api/firstfactor","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":266,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_firstfactor.go","Line":54,"Name":"FirstFactorPOST.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/bridge.go","Line":54,"Name":"(*BridgeBuilder).Build.func1.1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/headers.go","Line":25,"Name":"SecurityHeadersCSPNone.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/headers.go","Line":35,"Name":"SecurityHeadersNoStore.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/headers.go","Line":16,"Name":"SecurityHeaders.func1"},{"File":"github.com/fasthttp/router@v1.4.14/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/valyala/fasthttp@v1.43.0/http.go","Line":154,"Name":"(*Response).StatusCode"},{"File":"github.com/valyala/fasthttp@v1.43.0/server.go","Line":2338,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.43.0/workerpool.go","Line":224,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.43.0/workerpool.go","Line":196,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1594,"Name":"goexit"}],"time":"2024-01-24T12:01:53-05:00"}

**Comment by james-d-elliott** [comment, marked as answer]
Looks like a failure to perform a DNS lookup of the server: `"Network Error": dial tcp: lookup ForestDnsZones.corp.evilcorp.com: no such host`

**Comment by tltglitch** [comment]
@james-d-elliott  Thank you, this took me days to figure out but i found the issue.It was a DNS issue,  Authelia was trying to resolve my domain controllers host name on the internet instead of locally.

**Reply by james-d-elliott** [reply]
Yeah, Authelia will respect the system resolution order so if you change your system resolution settings this is not unexpected.
